### ISSUE TITLE
Server down?

### ISSUE THREAD

**Issue by gtoffoli** [up2university/kuboxed#37]
Hi, I've tried to access my SWAN account on the production server today, a few weeks after my previous access. The system hanged during the startup procedure (selection of kernel and configuration script).
Now I get the screen message "503 : Service Unavailable - Your server appears to be down. Try restarting it from the hub".

**Comment by diocas** [comment]
@gtoffoli We have to clean your account state. In the meantime, can you use the test version?

@michzimny can you have a look? You probably need to remove his container and go inside the swan/jupyterhub pod to restart jupyterhub (`supervisorctl restart jupyterhub`). This problem should be fixed with the updated version I sent you recently.

**Comment by michzimny** [comment]
I haven't checked yet, but I will do the upgrade soon today.

**Comment by diocas** [comment]
Ok, when you upgrade, please remove all the user containers. And you can also delete the jupyterhub.sqlite.

**Comment by michzimny** [comment]
In the meantime, I've resterted jupyterhub as @diocas suggested. It led to some other errors, so then I simply recreated the jupyterhub pod. It seems to be working for me know.

@gtoffoli, please check if it's fine for you know.

The test instance is now upgraded. The production instance is not yet (but seems to be fixed).

**Comment by gtoffoli** [comment]
Now they are both working fine for me! Many thanks!

**Comment by gtoffoli** [comment]
The server is not responding. Can you have a look? Thanks.

**Comment by michzimny** [comment]
It's not CERNBox, it's Up2U SSO that is unavailable. You are not authenticated, so CERNBox and SWAN try to redirect you to the SSO.

See here:
```
michzimny@Michzimny-PCSS-T480:~$ curl https://cernbox.up2university.eu/ -D /dev/stdout
HTTP/1.1 301 Moved Permanently
Server: nginx/1.6.2
Date: Fri, 07 Feb 2020 16:16:28 GMT
Content-Type: text/html
Content-Length: 184
Connection: keep-alive
Location: https://cernbox.up2university.eu:443/cernbox

(...)
michzimny@Michzimny-PCSS-T480:~$ curl https://cernbox.up2university.eu/cernbox -D /dev/stdout
HTTP/1.1 302 Found
Server: nginx/1.6.2
Date: Fri, 07 Feb 2020 16:16:39 GMT
Content-Type: text/html; charset=iso-8859-1
Content-Length: 814
Connection: keep-alive
Expires: Wed, 01 Jan 1997 12:00:00 GMT
Cache-Control: private,no-store,no-cache,max-age=0
Location: https://sso.up2university.eu/simplesaml/saml2/idp/SSOService.php?xxx

(...)
```

I just opened ticket in SSO repo: https://github.com/up2university/single-sign-on/issues/72

**Comment by gtoffoli** [comment]
Trying to enter into my SWAN account, I get the "500 : Internal Server Error" message  at the address https://swan.up2university.eu/hub/user/up001009/
CernBox is ok.

**Comment by michzimny** [comment]
I'm having the same. Solving this one is currently blocked by #43.

**Comment by michzimny** [comment]
Closing as (currently) duplicate of #44.
